lib/node_modules/@stdlib/random/base/mt19937/docs/repl.txt

```diff
@@ -0,0 +1,94 @@
+
+{{alias}}()
+    Returns a pseudorandom integer on the interval `[1, 2147483646]`.
+
```

Comment:
Do we know the period of MT19937? I believe this is documented somewhere. And if so, we should include the period in the documentation, both here and in the README, as done for `minstd`.

Reply:
Yes, I put the period of MT19937 in this file. Not sure why this is still showing as changes requested.